daangn.css

```
@@ -0,0 +1,161 @@
+/* footer body */
+#footer_root {
+  border-top: 1px solid lightgray;
+  padding: 0 1.6rem;
+  box-sizing: border-box;
+  width: 100%;
+  max-width: 100vw;
+}
+.footer_body {
+  display: flex;
+  flex-direction: column;
+  max-width: 50rem;
+  margin: 0 auto;
+}
+
+/* footer section 1 */
+.footer_section1 {
+  padding: 4.8rem 0;
+  display: block;
+}
+.footer_menus {
+  display: flex;
+  max-width: 76.8rem;
+  font-size: 0.8rem;
+}
+.footer_menus_row {
+  display: flex;
+  flex-direction: column;
+  padding-inline-start: 0;
+  margin-block-start: 0;
+  margin-block-end: 0;
+  width: 33%;
+  list-style-type: disc;
+}
+.footer_menus_item:not(:last-child) {
+  margin-bottom: 1.6rem;
+}
+.footer_menus_item {
```

원본 웹사이트처럼 마우스를 올렸을 때 검정색 밑줄이 쳐지게 해도 좋을 것 같아요 !
```suggestion
.footer_menus_item a:hover {
  text-decoration: underline;
}
```

이런 식으로 ':' 뒤에 액션 (hover, mouseon, mouseout...등등)을 적어주고 중괄호 안에 그때 적용될 css 속성을 적어주면 됩니다!